CONTRIBUTING.adoc

```diff
@@ -50,101 +45,98 @@ https://github.com/openshift/origin[origin repo].
 === Develop locally on your host
 
 You can develop OpenShift 3 on Windows, Mac, or Linux, but you'll need Docker
-installed on Linux to actually launch containers.
+installed on Linux to actually launch containers. Client and server binaries
+can be built locally or in the `openshift/release` container environemnt. The
+http://golang.org/[Go] programming language is only necessary for building on
+the local host.
```

Comment:
@stevekuznetsov nothing is missing, just run the build script(or "go install -race std" prior) as root or set -pkgdir in to writable location
*-race subpackage is containing all the pre-built bits. It is just convenient(consistent,...) way to do it. And I think that installing it by default(it is few M) is not good as you not always need it.
Thanks for the env var, will try.

@adelton I will also add in to f25(it will take few day to bubble in to stable/updates repo)